c++/src/Reader.cc

```diff
@@ -1389,6 +1389,13 @@ namespace orc {
       contents->footer = REDUNDANT_MOVE(readFooter(stream.get(), buffer.get(),
         footerOffset, *contents->postscript,  *contents->pool));
     }
+    if (contents->postscript->version_size() != 2) {
+      contents->fileVersion = std::unique_ptr<FileVersion>(new FileVersion(0, 11));
```

Comment:
Can this statement 'new FileVersion(0,11)' be changed to 'new FileVersion(FileVersion::v_0_11())'.

Reply:
Yes. BTW, I refactored the codes to just keep an `isDecimalAsLong` flag in `FileContents` so this can be simpler.